Mathlib/Order/Directed.lean

```diff
@@ -63,10 +63,17 @@ theorem directedOn_iff_directed {s} : @DirectedOn α r s ↔ Directed r (Subtype
 alias directedOn_iff_directed ↔ DirectedOn.directed_val _
 #align directed_on.directed_coe DirectedOn.directed_val
 
-theorem directedOn_range {f : β → α} : Directed r f ↔ DirectedOn r (Set.range f) := by
+theorem directedOn_range {f : ι → α} : Directed r f ↔ DirectedOn r (Set.range f) := by
   simp_rw [Directed, DirectedOn, Set.forall_range_iff, Set.exists_range_iff]
 #align directed_on_range directedOn_range
 
+-- porting note: This lemma was misplaced in mathlib
```

Comment:
What do you mean by this?

Reply:
The `alias` was declared in `order.compactly_generated`

Reply:
```suggestion
-- porting note: This alias was misplaced in `order/compactly_generated.lean` in mathlib3
```